tests/h/search/query_test.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 import webob
 
+from h.search.index import delete
 from h.search import Search, query
```

Comment:
Could we remove this line and in the following line import `index` also? So: `from h.search import Search, query, index`

Reply:
Done. I was originally trying to avoid confusion with the fixture named `index` but I see we aren't using it in these tests.